README.md

```diff
@@ -7,21 +7,22 @@ This repository holds the 5GMS Application Server implementation for the 5G-MAG
 The 5GMS application server (AS) is a Network Function that forms part of the 5G Media Services framework as defined in
 ETSI TS 126.501.
 
-This implementation is comprised of a small Python daemon process which implements the 5GMS M3 interface as a simple
-configuration file that is shared with the Reference
-Tools [5GMS AF](https://github.com/5G-MAG/rt-5gms-application-function).
+This implementation is comprised of a small Python daemon process which implements the 5GMS M3 interface and controls an external
```

Comment:
How about "...a small Python process which implements the 5GMS AS configuration service at interface M3, and which also manages an HTTP(S) daemon subprocess to ingest content at interface M2d and serve it to 5GMS Clients at interface M4d."

Reply:
Done